app/mailers/participant_validation_mailer.rb

```diff
@@ -1,63 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Is this the right PR to remove it?

Even if we think it's a good enough PR to remove it... can we mention it in the description?

Reply:
Description updated - this whole mailer was a duplicate of PariticpantMailer, sending same templates with quite a different personalisation. It took me a while to figure this out, so decided to remove it in this PR, especially given that most of the tags added in this PR have no backing in required features making this PR pure TechDebt